airgun/entities/repository.py

```diff
@@ -49,6 +53,46 @@ def synchronize(self, product_name, entity_name):
         view.progressbar.wait_for_result()
         return view.read()
 
+    def remove_packages(self, product_name, entity_name):
+        """Remove all packages from repository"""
+        view = self.navigate_to(
+            self,
+            'Packages',
+            product_name=product_name,
+            entity_name=entity_name,
+        )
+        view.items_per_page.fill('100')
```

Comment:
maybe we can select latest value in the list, not the static `100`? or maybe no =)

Reply:
latest is probably not the safest idea. I can see another option - fetch all the values, convert to int, find max, select it (and use that value in range of `for` loop). But that may be a bit overcomplicated. Thoughts?

Reply:
yeah, that was my initial idea, but probably it doesn't worth to spend efforts on that